pwnlib/shellcraft/templates/amd64/pushstr.asm

```diff
@@ -22,12 +21,11 @@ Args:
         extend = '\xff'
     else:
         extend = '\x00'
-%>
-
+%>\
```

Comment:
Actually exactly that change was not necessary, as the beginnings and ends are stripped as we talked about.